lib/resty/core.lua

```diff
@@ -17,6 +17,10 @@ if subsystem == 'http' then
     require "resty.core.response"
     require "resty.core.time"
     require "resty.core.worker"
+
+    local ngx_re = require "ngx.re"
+    ngx.re.split = ngx_re.split
+    ngx.re.opt = ngx_re.opt
```

Comment:
Actually we want to retire the `ngx.re.xxx` way at some point in the future. This is because for applications that do not need `ngx.re`, there is currently no way to avoid its GC overhead and memory overhead.

The future is to modularize the Lua API instead of growing the flat API namespace and tables forever.